### PR TITLE
KEP 3157 (watch-list): add a new beta criterion for client libraries

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: keps/sig-api-machinery/3157-watch-list/README.md [hunk 1]
```diff
@@ -682,6 +682,9 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Use WatchProgressRequester to request progress notifications directly from etcd.
   This mechanism was developed in [Consistent Reads from Cache KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache#use-requestprogress-to-enable-automatic-watch-updates)
   and will reduce the overall latency for watchlist requests.
+- The watchlist call, which serves as a drop-in replacement for list calls in client libraries,
```
Comment (deads2k) [inline keps/sig-api-machinery/3157-watch-list/README.md:685]: Perhaps add a direct reference to https://github.com/kubernetes/kubernetes/pull/126191 so future-us will have a reminder of exactly what broke.

Comment (p0lyn0mial) [reply keps/sig-api-machinery/3157-watch-list/README.md:685]: added the link, ptal.

**Pull request merged**